ambuda/static/js/reader.js

```diff
@@ -246,9 +146,9 @@ export default () => ({
     // Sync UI with application state. See comments on `uiScript` for details.
     this.uiScript = this.script;
 
-    // Load legacy content.
-    Dictionary.init();
-    ParseLayer.init();
+    // Allow sidebar to be shown. (We add `hidden` by default so that the
+    // sidebar doesn't display while JS is loading.)
+    $('#sidebar').classList.remove('hidden');
```

Comment:
Should we instead just use `showSidebar`?

Reply:
I'm not sure what Alpine's mechanism is for show/hide, but it interacts poorly if the element classlist already has display:none set. I'll update the comment to explain this.